INN/umbrella-citylimits
# Update Grunt, remove unnecessary items from Gruntfile, register task 'default'

## Changes

This pull request makes the following changes:

- Updates grunt dependencies
- Drops `package.lock`
- Removes the `version` task, because we haven't been versioning this software for release
- Drops `grunt-contrib-pot`, `grunt-confirm`, `grunt-git`, `grunt-po2mo`, `grunt-pot`, and `grunt-version` as they're unused in this project. If we end up making this theme translatable, we should use `grunt-shell` as a wrapper for the `wp i18n` tools.

## Why

This should clear up the [npm-related security alerts for this repo](https://github.com/INN/umbrella-citylimits/network/alerts), as well as bring improved syntax handling in LESS files for CSS' `calc()`.

## Testing/Questions

Features that this PR affects:

- CSS/LESS compilation in dev mode


Questions that need to be answered before merging:


Steps to test this PR:

1. the CSS loads, right?
2. Run `grunt` and watch for errors: there should be none, and the `default` task should be run.
